Nonprofit-Exchange-Hub/web-app
# new favicon

## Dev Summary
got new assets from design 😍 


## Test Plan

repro steps:

1. start up frontend
2. see new coolness
![image](https://user-images.githubusercontent.com/16400333/198056819-fd5f23c9-e2e7-498e-88c1-67fa559b417c.png)
